compiler/modules/front/symbols.bal

```diff
@@ -11,6 +12,14 @@ type ModuleSymbols record {|
     map<Import>[] partPrefixes = [];
     t:Context tc;
     boolean allowAllTypes = false;
+    ResolveType[] deferredEmptinessChecks = [];
+    map<[boolean, d:Location]> recursiveEmptinessChecked = {};
```

Comment:
Use a record with meaningful names, not a tuple.

Reply:
Fixed